include/ensmallen_bits/nsga2/nsga2_impl.hpp

```diff
@@ -396,9 +398,7 @@ inline bool NSGA2::Dominates(
 inline void NSGA2::CrowdingDistanceAssignment(const std::vector<size_t>& front,
                                               std::vector<double>& crowdingDistance)
 {
-  if (front.size() > 0)
-  {
-    for (size_t elem: front)
+    for (size_t elem: front) //TODO: When looping over all the fronts, all will be initialized to zero right?
       crowdingDistance[elem] = 0;
```

Comment:
Since we're looping over all the fronts, can we not simply initialize ```crowdDistance``` ahead of calling this function?